docs/src/snforge-advanced-features/storage-cheatcodes.md

```diff
@@ -7,7 +7,8 @@ which allow manipulating the storage directly (reading and writing).
 In order to obtain the variable address that you'd like to write to, or read from, you need to use either:
 - `selector!` macro - if the variable is not a mapping
 - `map_entry_address` function in tandem with `selector!` - for key-value pair of a map variable
-
+- `starknet::storage_access::storage_address_from_base`
```

Comment:
I think we may want to move it as the first method, since it is the most convenient, cc @Arcticae

Reply:
Not really since it requires you to have the source code. 
Say if you have erc-20, and you want to change the balance/admin - you won't be able to do that easily without the source (which is oftentimes the case)